vod/hls/m3u8_builder.c

```diff
@@ -511,48 +511,56 @@ m3u8_builder_build_master_playlist(
 	media_set_t* media_set,
 	vod_str_t* result)
 {
-	media_track_t** cur_sequence_tracks;
-	media_sequence_t* cur_sequence;
-	media_track_t* track;
+	media_sequence_t* cur_sequence;	
 	media_info_t* video;
 	media_info_t* audio = NULL;
 	uint32_t sequence_index;
 	uint32_t bitrate;
 	u_char* p;
-	size_t max_video_stream_inf;
-	size_t result_size;
+	size_t          max_video_stream_inf = 0;
+	size_t          result_size = 0;
+        uint32_t        main_media_type;
```

Comment:
please change the indentation to tab, and remove the multiple spaces that prefix the variable name, the convention is to use a single space. you can search your updated file for '    ' (4 spaces) to make sure you always use tabs